app/models/tag.rb

```diff
@@ -204,4 +204,14 @@ def followers_who_dont_follow_tags(tags)
     following_given_tags = User.where(id: uids)
     tag_followers.reject { |user| following_given_tags.include? user  }
   end
+
+  def self.trending(limit = 5, time_ago = 1.month)
```

Comment:
This is great. I'd like to suggest we modify this to be `trending(limit = 5, start = Time.now, end = Time.now - 1.month)` -- that way we can use this method for any range of time. What do you think?

And, i'd like to suggest we test this method in a unit test, in `/test/unit/tag_test.rb` -- it should be pretty simple, what do you say?

Reply:
Yes sir...I also think that self.trending method should accept range arguments .